### PR TITLE
change email regex

### PR DESCRIPTION
**Pull request by jinnnn9**
- fix email regex so it does not break build

### CODE REVIEW

File: src/foam/nanos/u2/navigation/SignIn.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ foam.CLASS({
       hidden: true
     },
     {
-      class: 'EMail',
+      class: 'String',
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/SignIn.js]: Why the change back to String?

Comment (jinnnn9) [reply src/foam/nanos/u2/navigation/SignIn.js]: I remembered someone had mentioned not wanting to show validation when users sign up. If we set class to EMail, it will display validation text so I reverted it back to how it had been.

Comment (TW80000) [reply src/foam/nanos/u2/navigation/SignIn.js]: Could you leave it as Email but override the view for it in the sign in form instead?

Comment (jinnnn9) [reply src/foam/nanos/u2/navigation/SignIn.js]: I think this is the reason why someone who wrote this code set class to String in the first place.

Comment (kgrgreer) [reply src/foam/nanos/u2/navigation/SignIn.js]: Yes, why?

Comment (jinnnn9) [reply src/foam/nanos/u2/navigation/SignIn.js]: I think you wouldn't usually have validation text display on login page (I think it was the writer's intention). @TW80000 I will try your approach.

File: src/foam/nanos/u2/navigation/SignIn.js [hunk 2]
```diff
@@ -33,13 +33,11 @@ foam.CLASS({
...
       },
-      preSet: function(o, n) {
-        return n.trim();
-      },
       visibilityExpression: function(disableEmail_) {
         return disableEmail_ ?
           foam.u2.Visibility.DISABLED : foam.u2.Visibility.RW;
-      }
+      },
+      validationTextVisible: false
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/SignIn.js:40]: So this applies to sign-up as well?

Comment (jinnnn9) [reply src/foam/nanos/u2/navigation/SignIn.js:40]: I haven't looked into it. I will change signUp as well if you want me to. I can also check any other places that use EMail to make sure they work well.

Comment (jlhughes) [reply src/foam/nanos/u2/navigation/SignIn.js:40]: got it, thought signup and signin were the same model. So this is fine.  Thanks.

**Review by jlhughes — APPROVED** [review]
